src/braket/tasks/local_quantum_task.py

```diff
@@ -23,7 +23,10 @@ class LocalQuantumTask(QuantumTask):
     Since this class is instantiated with the results, cancel() and run_async() are unsupported.
     """
 
-    def __init__(self, result: Union[GateModelQuantumTaskResult, AnnealingQuantumTaskResult]):
+    def __init__(
```

Comment:
Interesting that tox did this

Reply:
Ah I think what actually happened here is originally I created a new result type and expanded the union to three fields, which made tox split the line; when I removed the new type tox found the split line still acceptable